test/assetGiveaway/assetGiveaway.test.ts

```diff
@@ -19,12 +19,12 @@ describe('Asset_Giveaway_1', function () {
     const asset = assets[0];
     const proof = tree.getProof(calculateAssetHash(asset));
     const giveawayContractAsUser = await giveawayContract.connect(
-      ethers.provider.getSigner(others[1])
+      ethers.provider.getSigner(others[0])
```

Comment:
Is this to tidy up or is there another reason not to use `others[1]`?

Reply:
+1 (I'm also using others[1] in my current branch)

Reply:
It is because of how the claim was setup, I did not change the data but had to change how the others was setup.
if I kept using others[1] I would have to change the claim data

but I think moving forward we should name all accounts in the fixtures to have clearer tests